appserver/website-bootstrap/main.css

```diff
@@ -382,105 +393,4 @@ body {
 	list-style-type: none;
 	float: left;
 	padding: 10px;
-}
-
-.team-members a {
-	color: inherit;
-}
-
-.team-members a:hover {
-	text-decoration: none;
-	color: #9191c4;
-}
-
-/* Temporal Page buttons */
-form.temporal-search input {
-  padding: 10px;
-  font-size: 17px;
-  border: 1px solid grey;
-  float: left;
-  width: 80%;
-  background-color:#cccccc;
-  color: #000000;
-  margin-top: 20%
-}
-
-form.temporal-search button {
-  float: left;
-  width: 19%;
-  padding: 9px;
-  background: #2196F3;	
-  color: white;
-  font-size: 17px;
-  border: 1px solid grey;
-  border-left: none;
-  cursor: pointer;
-  margin-top: 20%;
-}
-
-form.temporal-search button:hover {
-  background: #0b7dda;
-}
-
-#rangeslider-text{
-	font-size: 20px;
-	margin-bottom: 5px;
-	color: white;
-}
-
-.rangeslider{
-	width: 150%;
-	margin-top: 100%;
-}
-
-
-.slider {
-    -webkit-appearance: none;
-    background: #ffffff;
-	background-color:#cccccc;
-    width: 50%;
-    height: 5px;
-    opacity: 5;
-	cursor: pointer;
-
-
-   }
-
-.slider:hover {
-    opacity: 1;
-}
-
-.rangeslider-value {
-  font-size: 20px;
-  color: white;
-  margin-top: 15px;
-
-  }
-
-
-#add-trend-btn{
-	margin-top: 30%;
-	background:transparent;
-	cursor: pointer;
-	color: white;
-	font-size: 20px;
-	padding: 0;
-	border: none;
-
-}
-
-
-#update-graph-btn{
-	margin-top: 20%;
-	margin-left: 18%;
-	background-color: #fff;
-	padding-top: 8px;
-	padding-bottom: 8px;
-	padding-left: 20px;
-	padding-right: 20px;
-	font-size: 18px;
-	font-weight: 600;
-	font-color: #2e2e4d;
-	border-radius: 4px;
-	cursor: pointer;
```

Comment:
I think you need to merge into your branch the latest commits in master (from Mike's temporal UI input elements branch)

Reply:
should be good to go now, should I make another pull request?

Reply:
added another commit, containing this code